src/lib/rendererHelper.ts

```diff
@@ -35,6 +35,20 @@ export async function renderImage(actionContext: string, state: ActionState) {
 		image.onload = resolve;
 	});
 
+	let alertImage = document.createElement("img");
+	alertImage.crossOrigin = "anonymous";
+	alertImage.src = "http://localhost:5173/alert.png";
```

Comment:
http://localhost:5173 is only the Vite dev server and is not accessible in a release build or if a different port is being used

Reply:
What else should i use then? I tried using `getImage`, but that lead to CORS errors